lava-slave/Dockerfile

```diff
@@ -1,40 +1,4 @@
-FROM bitnami/minideb:stretch
-
-RUN apt-get update
-
-# e2fsprogs is for libguestfs
-RUN \
- echo 'lava-server   lava-server/instance-name string lava-slave-instance' | debconf-set-selections && \
- echo 'locales locales/locales_to_be_generated multiselect C.UTF-8 UTF-8, en_US.UTF-8 UTF-8 ' | debconf-set-selections && \
- echo 'locales locales/default_environment_locale select en_US.UTF-8' | debconf-set-selections && \
- DEBIAN_FRONTEND=noninteractive apt-get -y install \
- locales \
- vim \
- sudo \
- cu \
- conmux \
- python-setproctitle \
- tftpd-hpa \
- u-boot-tools \
- device-tree-compiler \
- qemu-system \
- qemu-system-arm \
- qemu-system-i386 \
- xnbd-server \
- e2fsprogs
-
-RUN if [ "$(uname -m)" = "x86_64" -o "$(uname -m)" = "x86" ] ;then apt-get -y install qemu-kvm ; fi
-
-# Uncomment the following for having the latest package (you need to comment all build-lava below)
-#RUN DEBIAN_FRONTEND=noninteractive apt-get -y install wget gnupg \
-# && wget http://images.validation.linaro.org/production-repo/production-repo.key.asc \
-# && apt-key add production-repo.key.asc \
-# && echo 'deb http://images.validation.linaro.org/production-repo/ stretch-backports main' > /etc/apt/sources.list.d/lava.list \
-# && apt-get clean && apt-get update
-#RUN DEBIAN_FRONTEND=noninteractive apt-get -y install lava-dispatcher
-
-COPY scripts/build-lava /
-RUN sh /build-lava lava-dispatcher 2018.4 2018.4-1
+FROM montjoie/lava-docker-slave-image:latest
```

Comment:
Let's use the baylibre docker hub for this for the final version.

Reply:
done